schema/deploy/shipping/views.sql

```diff
@@ -2288,15 +2286,31 @@ grant select
 
 create or replace view shipping.scan_enrollments_v1 as
 
+    with location_names as (
+        select
+            scale,
+            identifier,
+            case scale
+                when 'neighborhood_district' then details->>'name'
+                when 'puma' then details->>'NAMELSAD10'
+            end as geo_location_name
+        from warehouse.location
+        where scale in ('neighborhood_district', 'puma')
+        and hierarchy @> 'state => washington'
+    )
+
     select
         illness_questionnaire_date,
         scan_study_arm,
         priority_code,
         puma,
         neighborhood_district,
```

Comment:
This view is just for Formative's dashboards, right? I think with the new `geo_location_name` field, these `puma` and `neighborhood_district` fields aren't needed…?

Reply:
I kept the fields so that nothing on their end breaks when I merge/deploy this change.